content/en/apps/guides/forms/rdtoolkit-integration.md

```diff
@@ -0,0 +1,219 @@
+    });
+
+    if (!captureReport || !getField(captureReport, 'data.rdtoolkit_results')) {
+      return false;
+    }
+
+    const startTime = Math.max(addDays(dueDate, -event.start).getTime(), report.reported_date + 1);
+    const endTime = addDays(dueDate, event.end + 1).getTime();
+
+    return isFormArraySubmittedInWindow(contact.reports, ['rdtoolkit-capture'], startTime, endTime);
+  },
+  actions: [
+    {
+      type: 'report',
+      form: 'rdtoolkit-capture',
+      modifyContent: (content, contact, report) => {
+        content.rdtoolkit_session_id = getField(report, 'data.rdtoolkit_session_id'); // Copy rdtoolkit_session_id from provision form to capture form.
+      }
+    }
+  ],
+  events: [
+    {
+      id: 'rdtoolkit-capture-event',
+      start: 1,
+      end: 2,
+      days: 1
+    }
+  ]
+}
+```
+
+## Add translations
+Add translations by following the [CHT guide](https://docs.communityhealthtoolkit.org/apps/reference/translations/). The following is an example of translation labels:
+
+```
+report.rdtoolkit-provision.data = Test details
+report.rdtoolkit-provision.data.rdtoolkit_state = RD test status
+report.rdtoolkit-provision.data.rdtoolkit_time_resolved = Results available on
+report.rdtoolkit-provision.data.rdtoolkit_time_started = Started on
+report.rdtoolkit-provision.data.__patient_id = Patient ID
+report.rdtoolkit-provision.data.__patient_name = Patient name
+
+report.rdtoolkit-capture.data = Test details
+report.rdtoolkit-capture.data.rdtoolkit_image = Image
+report.rdtoolkit-capture.data.rdtoolkit_results_description = Results
+report.rdtoolkit-capture.data.rdtoolkit_state = RD test status
+report.rdtoolkit-capture.data.rdtoolkit_time_read = Results were captured on
+report.rdtoolkit-capture.data.rdtoolkit_time_resolved = Results were available on
+report.rdtoolkit-capture.data.rdtoolkit_time_started = Started on
+report.rdtoolkit-capture.data.__patient_id = Patient ID
+report.rdtoolkit-capture.data.__patient_name = Patient name
+mal_pf = Malaria P. falciparum
+mal_pf_neg = Pf negative
+mal_pf_pos = Pf positive
+mal_pv = Malaria P. vivax
+mal_pv_neg = Pv negative
+mal_pv_pos = Pv positive
+sars_cov2 = SARS-CoV-2
+sars_cov2_neg = COVID-19 negative
+sars_cov2_pos = COVID-19 positive
+universal_control_failure = Invalid - control failed
+```
+
+## Upload the assets
+Once the forms, the task and the translation are ready, use [medic-conf](https://github.com/medic/medic-conf) to upload these assets to the CHT Core instance.
+* [Build and upload translations](https://docs.communityhealthtoolkit.org/apps/reference/translations/#build)
+* [Convert and upload forms](https://docs.communityhealthtoolkit.org/apps/reference/forms/app/#build)
+* [Build and upload tasks](https://docs.communityhealthtoolkit.org/apps/reference/tasks/#build)
+
+Alternatively the assets can be uploaded by using the [App Manager app](https://docs.communityhealthtoolkit.org/apps/features/admin/).
```

Comment:
Again, worth fixing these to [avoid broken links](https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/#avoid-broken-links).